Code/Tools/ProjectManager/Platform/Windows/ProjectUtils_windows.cpp

```diff
@@ -186,19 +186,50 @@ namespace O3DE::ProjectManager
                     // First try <project-build-path>/bin/$<CONFIG> and if that path doesn't exist
                     // try <project-build-path>/bin/$<PLATFORM>/$<CONFIG>
                     buildConfigurationPath /= "bin";
-                    if (editorPath = (buildConfigurationPath / AZ_BUILD_CONFIGURATION_TYPE / "Editor").
+                    auto paths = AZStd::vector<AZ::IO::FixedMaxPath>{
+                        buildConfigurationPath / AZ_BUILD_CONFIGURATION_TYPE / "Editor",
+                        buildConfigurationPath / AZ_TRAIT_OS_PLATFORM_CODENAME / AZ_BUILD_CONFIGURATION_TYPE / "Editor"
+                    };
+
+                    // always try profile config because that is the default
+                    if (strcmp(AZ_BUILD_CONFIGURATION_TYPE, "profile") != 0)
+                    {
+                        paths.emplace_back(buildConfigurationPath / "profile" / "Editor");
+                        paths.emplace_back(buildConfigurationPath / AZ_TRAIT_OS_PLATFORM_CODENAME / "profile" / "Editor");
+                    }
+
+                    for (auto path : paths)
```

Comment:
This should be a non-const lvalue reference as there is no need to copy the FixedMaxPath.
```suggestion
                    for (auto& path : paths)
```